src/baldr/CMakeLists.txt

```diff
@@ -38,14 +38,10 @@ set(includes
      ${CMAKE_CURRENT_BINARY_DIR}/src/baldr
      )
 
-# treat date library as system
 set(system_includes 
-  ${VALHALLA_SOURCE_DIR}/third_party/date/include
-  $<$<BOOL:${WIN32}>:${VALHALLA_SOURCE_DIR}/third_party/dirent/include>
-  ${VALHALLA_SOURCE_DIR}/third_party/rapidjson/include)
-if(APPLE)
-  list(APPEND system_includes ${VALHALLA_SOURCE_DIR}/third_party/date/include/date)
-endif()
```

Comment:
I removed this in all CMakeLists.txt, `${VALHALLA_SOURCE_DIR}/third_party/date/include/date` is not even a valid path